src/test/groovy/org/prebid/server/functional/testcontainers/scaffolding/NetworkScaffolding.groovy

```diff
@@ -46,6 +47,21 @@ abstract class NetworkScaffolding {
                         .size()
     }
 
+    boolean checkRequestCount(int expectedCount, int pollTime = 1000) {
+        def frequencyPolling = 10
+        def isRequestReceived = false
+
+        for (int i = 0; i < pollTime; i += frequencyPolling) {
+            Thread.sleep(frequencyPolling)
+            def requestCount = getRequestCount()
+            if (requestCount == expectedCount) {
```

Comment:
What will happen if you received more requests than you anticipated? Please add a separate check with an exception.